app/src/main/java/org/oppia/app/utility/PackageManagerHelper.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.pm.PackageManager
+import javax.inject.Inject
+
+/** Extension functions for PackageManager. */
+@Inject
+lateinit var context: Context
+
+fun PackageManager.getVersionName(): String {
```

Comment:
Nit: add KDoc here & below. Feel free to resolve once addressed.

Reply:
Okay